src/map/utils/battleutils.cpp

```diff
@@ -5023,6 +5032,37 @@ namespace battleutils
         PChar->PClaimedMob = nullptr;
     }
 
+    // Checks to see if the mob has a damage cap value
+    // This is used for instances like Suttung, Antaeus, Crustacean Conundrum bcnm, Colonization reives
+    int32 DamageCapDistribution(int32 damage, CBattleEntity* PDefender)
```

Comment:
I should think of a better name for this function. Not sure what I was thinking here. `CheckDamageCap` maybe?

Reply:
I like verboseness: `CheckAndApplyDamageCap`

Reply:
Changed